server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -375,6 +375,9 @@ public boolean isMetadataField(String field) {
     }
 
     public boolean isMultiField(String field) {
+        if (runtimeMappings.containsKey(field)) {
```

Comment:
interesting, was this covered before through the isRuntimeField method?

Reply:
No, I think it was an oversight.  I'll add a YAML test to cover it.

Reply:
Would it make sense to have test coverage for this in a unit test? For instance SearchExecutionTests?